full/Dockerfile

```diff
@@ -1,192 +1,149 @@
-
-ENV NODE_PATH ~/.nvm/v$NODE_VERSION/lib/node_modules
-ENV PATH      ~/.nvm/versions/node/v$NODE_VERSION/bin:$PATH
-
-## Install GO
-ENV GO_VERSION 1.11.2
-ENV GOROOT=${HOME}/go
-ENV PATH $PATH:${GOROOT}/bin
-ENV GOPATH=${HOME}/go-packages
-ENV PATH $PATH:${GOPATH}/bin
-
-RUN curl -sS https://storage.googleapis.com/golang/go$GO_VERSION.linux-amd64.tar.gz | tar -C ${HOME} -xzf -
-RUN go get -u -v github.com/ramya-rao-a/go-outline
-RUN go get -u -v github.com/acroca/go-symbols
-RUN go get -u -v github.com/nsf/gocode
-RUN go get -u -v github.com/rogpeppe/godef
-RUN go get -u -v golang.org/x/tools/cmd/godoc
-RUN go get -u -v github.com/zmb3/gogetdoc
-RUN go get -u -v golang.org/x/lint/golint
-RUN go get -u -v github.com/fatih/gomodifytags
-RUN go get -u -v github.com/uudashr/gopkgs/cmd/gopkgs
-RUN go get -u -v golang.org/x/tools/cmd/gorename
-RUN go get -u -v sourcegraph.com/sqs/goreturns
-RUN go get -u -v github.com/cweill/gotests/...
-RUN go get -u -v golang.org/x/tools/cmd/guru
-RUN go get -u -v github.com/josharian/impl
-RUN go get -u -v github.com/haya14busa/goplay/cmd/goplay
-RUN go get -u -v github.com/davidrjenni/reftools/cmd/fillstruct
-
-# Keep the GO packages installed under /usr/local/go-packages but install all user packages to /workspace
-ENV GOPATH=/workspace:${GOPATH}
-ENV PATH $PATH:/workspace/bin
-
-# Install yarn
-USER root
-RUN curl -L https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
-
-RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
-    && apt-get update \
-    && apt-get install -y --no-install-recommends yarn \
-    && apt-get clean && rm -rf /var/cache/apt/* && rm -rf /var/lib/apt/lists/* && rm -rf /tmp/* \
-    && echo Installing Yarn newer than 1.10.1
-
-# Install Java & Maven
-RUN apt-get update \
-  && apt-get install -y software-properties-common \
-  && apt-get clean && rm -rf /var/cache/apt/* && rm -rf /var/lib/apt/lists/* && rm -rf /tmp/*
+    && apt-get install -yq \
+        asciidoctor \
+        bash-completion \
+        build-essential \
+        jq \
+        less \
+        llvm \
+        locales \
+        man-db \
+        nano \
+        software-properties-common \
+        sudo \
+    && locale-gen en_US.UTF-8 \
+    && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/*
+ENV LANG=en_US.UTF-8
+
+### Gitpod user ###
+RUN useradd -u 33333 -G sudo -md /home/gitpod -s /bin/bash -p gitpod gitpod \
+# passwordless sudo for users in the 'sudo' group
+    && sed -i.bkp -e 's/%sudo\s\+ALL=(ALL\(:ALL\)\?)\s\+ALL/%sudo ALL=NOPASSWD:ALL/g' /etc/sudoers
+ENV HOME=/home/gitpod
+WORKDIR $HOME
+# custom Bash prompt
+RUN { echo && echo "PS1='\[\e]0;\u \w\a\]\[\033[01;32m\]\u\[\033[00m\] \[\033[01;34m\]\w\[\033[00m\] \\\$ '" ; } >> .bashrc
+
+### C/C++ ###
+RUN curl -fsSL https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - \
+    && apt-add-repository -yu "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-6.0 main" \
```

Comment:
it looks like we could/should upgrade to cosmic here, too, since our base image is cosmic.
Does not need to be done in this PR, since it was outdated before.

see http://apt.llvm.org

Reply:
I tried and it didn't work?!

Reply:
(Probably because I didn't see there's two `bionic`s in the string :-) Changed it as you suggest, thanks!)